lib/runner/test-runners/cucumber.js

```diff
@@ -88,13 +89,27 @@ class CucumberSuite extends TestSuite {
     });
   }
 
+  shouldUseNightwatchFormatter(options) {
+    return ((!options.format && 
+      !this.argv.format && 
+      NightwatchEventHub.isAvailable) || 
+      options.format === 'nightwatch-format' || 
+      this.argv.format === 'nightwatch-format'
+    );
```

Comment:
I think there can be multiple formatters in cucumber

Reply:
Indeed, there can be multiple formatter but this check will be applied either `onEvent` function provided in `globals` or when someone explicitly specifies `nightwatch-format`.

Reply:
`format` could be an array instead of a value in case of multiple formatters